CMakeLists.txt

```diff
@@ -475,6 +476,16 @@ psibase_package(
     DEPENDS doc ${CMAKE_CURRENT_BINARY_DIR}/book/html/index.html
 )
 
+psibase_package(
+    OUTPUT ${SERVICE_DIR}/NopSys.psi
```

Comment:
I was accidentally on a slightly older version of your package-update branch, and boot failed since this wasn't installed, but that's not happening anymore on the latest.

Reply:
fixed